packages/grid/test/dom/__snapshots__/grid.test.snap.js

```diff
@@ -213,7 +213,7 @@ snapshots["vaadin-grid shadow default"] =
           part="cell footer-cell first-column-cell first-footer-row-cell last-footer-row-cell"
           role="gridcell"
           style="width: 100px; flex-grow: 1; order: 10000000;"
-          tabindex="-1"
+          tabindex="0"
```

Comment:
Why is `tabindex` modified? Should we add `await nextRender()` in the snapshots?

Reply:
I added an explanation to the commit message

Reply:
Thanks, I missed it. Makes sense 👍